can/io/logger.py

```diff
@@ -208,17 +208,17 @@ def on_message_received(self, msg: Message) -> None:
         self.writer.on_message_received(msg)
 
     def _get_new_writer(self, filename: StringPathLike) -> FileIOMessageWriter:
-        """Instantiate a new writer after stopping the old one.
+        """Instantiate a new writer.
+
+        .. note::
+            The :attr:`self.writer` should be closed prior to calling this function.
```

Comment:
I removed "after stopping the old one" from the doc string and also added in a note that writer should be closed prior to calling the function.